editor/metaboxes/index.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { isEqual } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, findDOMNode } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { handleMetaboxReload, metaboxStateChanged } from '../actions';
+
+// @TODO add error handling.
+class Metaboxes extends Component {
+	constructor() {
+		super();
```

Comment:
can we use `super( ...arguments )` here?

Reply:
I vaguely recall the linter complaining about that for some reason, but we can definitely do that.